ChessStatePattern/src/states/commons/Move.java

```diff
@@ -0,0 +1,26 @@
+package states.commons;
+
+public class Move {
+    private int row;
+    private int col;
+    public Move(int row, int col) {
```

Comment:
Just a discuss that
I was thinking if there need to set up edges condition and throw exception due to the board is 8 X 8
it's like once  0 > row or 7 < row or 0 > col or 7 < col